packages/cli/src/utils/platform.ts

```diff
@@ -15,19 +15,16 @@ export const platformParameters = {
     // TODO Remove this from global once we have a way for parents to strip out flags upon parsing.
     '--early-access-feature': Boolean,
     '--token': String,
-    '-t': '--token',
```

Comment:
Huh, this is unexpected. Why?

Reply:
Ah, so Prisma CLI or Checkpoint does not support short options yet?

If so, we should create an issue because that is quite a DX thing to have to drop. Maybe @sampolahtinen can do a PR on checkpoint as well.